mmrotate/models/detectors/rotated_fcos.py

```diff
@@ -1,22 +1,30 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-from ..builder import ROTATED_DETECTORS
-from .single_stage import RotatedSingleStageDetector
+from mmdet.models.detectors.single_stage import SingleStageDetector
+from mmdet.utils import ConfigType, OptConfigType, OptMultiConfig
 
+from mmrotate.registry import MODELS
 
-@ROTATED_DETECTORS.register_module()
-class RotatedFCOS(RotatedSingleStageDetector):
+
+@MODELS.register_module()
+class RotatedFCOS(SingleStageDetector):
```

Comment:
Do we still need this class?

Reply:
THe configs are using "mmdet.FCOS"

Reply:
This class is useless, should we remove that?